db/migrate/20160426212643_fix_step_user_completer.rb

```diff
@@ -0,0 +1,36 @@
+class FixStepUserCompleter < ActiveRecord::Migration
+  class Step < ActiveRecord::Base
+    belongs_to :user
+    belongs_to :completer, class_name: "User"
+  end
+
+  class User < ActiveRecord::Base
+    has_many :outgoing_delegations, class_name: "UserDelegate", foreign_key: "assigner_id"
+    has_many :incoming_delegations, class_name: "UserDelegate", foreign_key: "assignee_id"
+  end
+
+  class UserDelegate < ActiveRecord::Base
+    belongs_to :assignee, class_name: "User", foreign_key: "assignee_id"
+    belongs_to :assigner, class_name: "User", foreign_key: "assigner_id"
+  end
+
+  def up
+    steps_with_no_completer.each do |step|
```

Comment:
I suppose it could. It really should have been part of the migration in 04fea6b05913aa543140d64405092ef99d5fb443 (data migration to accompany schema change) but we failed to do that (didn't seem necessary at the time).

Plus, I want it to run once in every env, automatically. Which is what every db migration does.